ci-operator/config/openshift/cluster-api-provider-ibmcloud/openshift-cluster-api-provider-ibmcloud-release-4.9.yaml

```diff
@@ -0,0 +1,94 @@
+base_images:
+  base:
+    name: "4.9"
+    namespace: ocp
+    tag: base
+  ocp_builder_rhel-8-golang-1.15-openshift-4.6:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.15-openshift-4.6
+  ocp_builder_rhel-8-golang-1.16-openshift-4.8:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.16-openshift-4.8
+  ocp_builder_rhel-8-golang-openshift-4.6:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-openshift-4.6
+  openshift_release_golang-1.13:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+binary_build_commands: NO_DOCKER=1 make build
+build_root:
+  from_repository: true
+canonical_go_repository: sigs.k8s.io/cluster-api-provider-ibmcloud
+images:
+- dockerfile_path: Dockerfile.rhel
+  from: base
+  inputs:
+    ocp_builder_rhel-8-golang-1.16-openshift-4.8:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.8
+  to: ibmcloud-machine-controllers
+promotion:
+  disabled: true
+  name: "4.9"
+  namespace: ocp
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.9"
+  namespace: ocp
+tests:
+- as: unit
+  commands: NO_DOCKER=1 make test
+  container:
+    from: src
+- as: goimports
+  commands: IS_CONTAINER=TRUE make goimports
+  container:
+    from: src
+- as: govet
+  commands: IS_CONTAINER=TRUE make vet
+  container:
+    from: src
+- as: generate
+  commands: IS_CONTAINER=TRUE NO_DOCKER=1 make generate
+  container:
+    from: src
+- as: golint
+  commands: IS_CONTAINER=TRUE make lint
+  container:
+    from: src
+- as: e2e-ibmcloud
+  steps:
+    cluster_profile: ibmcloud
+    workflow: openshift-e2e-ibmcloud
+- as: e2e-ibmcloud-operator
+  steps:
+    cluster_profile: ibmcloud
+    test:
+    - as: test
+      cli: latest
+      commands: JUNIT_DIR=${ARTIFACT_DIR} make test-e2e
+      from: stable:cluster-api-actuator-pkg
+      resources:
+        requests:
+          cpu: 100m
+    workflow: ipi-ibmcloud
+- as: e2e-ibmcloud-upgrade
+  steps:
+    cluster_profile: ibmcloud
+    workflow: openshift-upgrade-ibmcloud
+- as: e2e-ibmcloud-serial
+  steps:
+    cluster_profile: ibmcloud
+    workflow: openshift-e2e-ibmcloud-serial
```

Comment:
given the discussion about using the aws-e2e to get things going, should these be trimmed back for now? (eg replaced with just the aws-e2e sanity check)

Reply:
Replace the code referenced above with the following ... ?
```
- as: e2e-aws
  steps:
    cluster_profile: aws
    workflow: openshift-e2e-aws
```

Reply:
I think so. We won't have any `upgrade` or `serial` tests to start with for ibmcloud e2e.
I think just the `e2e-aws` tests would be sufficient until we can add and expand ibmcloud CI workflows

Reply:
agreed, i think this is fine for the first pass just to make sure we aren't disrupting the status quo. then we can layer the ibm cloud specific tests on as we create them and get the infra setup.

Reply:
done